Assignment3/Q1_test.py

```diff
@@ -0,0 +1,49 @@
+"""Tests for question 1 - word search."""
+
+import unittest
+
+import Q1
+
+
+class Q1Test(unittest.TestCase):
```

Comment:
I did what you suggested here. I just couldn't get the ```*words``` argument to work, I think it was creating a tuple not a list and then word_list[i].upper() (in Dictionary) was giving me an error. So I am passing a list ```words``` as argument now. Couldn't figure out how to fix ```*words``` version.